zed_wrapper/src/nodelet/src/zed_wrapper_nodelet.cpp

```diff
@@ -1051,6 +1062,16 @@ namespace zed_wrapper {
     }
 
     void ZEDWrapperNodelet::publishPointCloud() {
+        // >>>>> Publish freq calculation
```

Comment:
I strongly associate ">>>>>" with "something went wrong when merging" :wink: 
I think it's best to avoid it to prevent confusion